include/pando-lib-galois/import/ingest_rmat_el.hpp

```diff
@@ -91,18 +91,12 @@ galois::DistLocalCSR<VertexType, EdgeType> initializeELDLCSR(pando::Array<char>
 
 #ifdef FREE
   galois::WaitGroup freeWaiter;
+  PANDO_CHECK(freeWaiter.initialize(0));
```

Comment:
thank goodness it was disabled by default

Reply:
lol.